rerun_py/src/remote.rs

```diff
@@ -1,3 +1,6 @@
+// TODO(rerun-io/dataplatform#405), TODO(rerun-io/dataplatform#400): remove this (now disabled) file
+// when the API migration is completed.
+
```

Comment:
I've excluded this file from the project but I think keping it "as reference" for the duration of the migration is useful.